addons/grenades/CfgAmmo.hpp

```diff
@@ -124,4 +124,26 @@ class CfgAmmo {
         soundImpactWoodExt3[] = {"A3\sounds_f\characters\footsteps\wood_run_ext_2",1,1.1,75};
         soundImpactWoodExt4[] = {"A3\sounds_f\characters\footsteps\wood_run_ext_2",1,1.15,75};
     };
+
+    class ACE_G_Incendiary: SmokeShell {
```

Comment:
Tbh, I don't like these military designations as display names. It's confusing.
Best example is `XM307` vs `XM312`. I can never tell which one is which. (we should probably change the names of those)